lib/mixins/text.coffee

```diff
@@ -185,7 +185,30 @@ module.exports =
           textWidth = @widthOfString(text.replace(/\s+/g, ''), options)
           spaceWidth = @widthOfString(' ') + characterSpacing
           wordSpacing = Math.max 0, (options.lineWidth - textWidth) / Math.max(1, words.length - 1) - spaceWidth
-          
+
+    # text baseline alignments based on http://wiki.apache.org/xmlgraphics-fop/LineLayout/AlignmentHandling
+    if typeof options.baseline is 'number'
+      dy = -options.baseline
+    else
+      switch options.baseline
+        when 'middle'
+          dy = 0.5 * @_font.xHeight
+        when 'central'
+          dy = 0.5 * (@_font.descender + @_font.ascender)
+        when 'after-edge'
+          dy = @_font.descender
+        when 'baseline'
+          dy = 0;
+        when 'mathematical'
+          dy = 0.5 * @_font.ascender
+        when 'hanging'
+          dy = 0.8 * @_font.ascender
+        when 'before-edge'
+          dy = @_font.ascender
+        else
+          dy = @_font.ascender
+      dy = dy / 1000 * @_fontSize
```

Comment:
Cool! Can we make these match the canvas `textBaseline` property values? https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/textBaseline